components/dashboard/src/data/workspaces/list-workspaces-query.ts

```diff
@@ -57,3 +57,29 @@ export function getListWorkspacesQueryKey(orgId?: string) {
     }
     return ["workspaces", "list", orgId];
 }
+
+export function useUpdateWorkspaceInCache() {
```

Comment:
it does not seem to be really hit by anything at runtime? :thinking:

Reply:
is it just optimization? dashboard already watching all wroskapces, do we really need this refactoring?

Reply:
Yes, I found no where right now in dashboard could use it after I add it (start and create are using React Class, which can't use use-query). Only add some TODOs for it

Reply:
But yes I will remove it then

Reply:
Let's create an issue for it and add to EXP inbox? You can live a code pointer for git history?